src/components/SingleResultPage/SingleResultPage.js

```diff
@@ -52,20 +54,20 @@ const SingleResultPage = ({ business, user, addFavorite, deleteFavorite }) => {
   return(
     <section className="single-result-section">
```

Comment:
We may need to make some adjustments to styling of `<section className="single-result-section">`, `<article className="single-result">` and/or `<div className="single-title-container">` - the some of the text lands outside of the container on larger screens

<img width="1558" alt="Screenshot 2022-12-14 at 9 37 51 AM" src="https://user-images.githubusercontent.com/100455148/207659162-66ffbc37-5ee2-414b-a182-463bfc8eefa3.png">